templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -9,5 +9,8 @@ <h1>Delete Manufacturer</h1>
   <form action="" method="post">
     {% csrf_token %}
     <input type="submit" value="Yes, delete" class="btn btn-danger">
+    <a href="{% url 'taxi:manufacturer-list' %}" class="btn btn-primary">

```

Comment:
leed it to the previous page, it will be more user-friendly

Reply:
I don`t have manufacturer detail